src/databricks/labs/remorph/resources/assessments/synapse/utilization/query_stats.sql

```diff
@@ -0,0 +1,3 @@
+--- Query Stats
+
+select *, CURRENT_TIMESTAMP as extract_ts FROM SYS.DM_PDW_NODES_EXEC_QUERY_STATS
```

Comment:
Queries look great and make logical sense. One small nit - can we run the queries through a formatter to ensure consistent casing, indentation, etc.?

Reply:
Sure I will and add it as part of our CI.